build_web.sh

```diff
@@ -1,77 +0,0 @@
-#!/usr/bin/env bash
-set -eu
-
-CRATE_NAME=${PWD##*/} # assume crate name is the same as the folder name
-
-script_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
-cd "$script_path"
-
-OPEN=false
-OPTIMIZE=false
-
-while test $# -gt 0; do
-  case "$1" in
-    -h|--help)
-      echo "build_web.sh [--optimize] [--open]"
-      echo "  --optimize: enable optimization step"
-      echo "  --open: open the result in a browser"
-      exit 0
-      ;;
-    -O|--optimize)
-      shift
-      OPTIMIZE=true
-      ;;
-    --open)
-      shift
-      OPEN=true
-      ;;
-    *)
-      break
-      ;;
-  esac
-done
-
-./setup_web.sh
-
-CRATE_NAME_SNAKE_CASE="${CRATE_NAME//-/_}" # for those who name crates with-kebab-case
-
-# This is required to enable the web_sys clipboard API which egui_web uses
-# https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.Clipboard.html
-# https://rustwasm.github.io/docs/wasm-bindgen/web-sys/unstable-apis.html
-export RUSTFLAGS=--cfg=web_sys_unstable_apis
```

Comment:
This was forgotten when porting to `trunk`, so copying text doesn't work!

This can be fixed with `RUSTFLAGS="--cfg=web_sys_unstable_apis" trunk serve` but it would be nice if `trunk` could add those build-flags automatically.

Reply:
@coderedart any idea how to tell `trunk` to always add `RUSTFLAGS="--cfg=web_sys_unstable_apis"` ?